content/en/docs/reference/glossary/manifest.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Manifest
+id: manifest
+date: 2019-06-28
+full_link: /docs/user-journeys/users/application-developer/foundational/#section-2
```

Comment:
I think I would omit this link rather than hyperlink to that page, to be honest.

It doesn't seem like it's a stable link, in as much as the contents of `#section-2` might be something else in the future.

Reply:
I have made necessary update in the commit below.